src/main/java/com/scottescue/dropwizard/entitymanager/TemporaryClassLoader.java

```diff
@@ -0,0 +1,156 @@
+package com.scottescue.dropwizard.entitymanager;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.hibernate.engine.jdbc.StreamUtils;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+class TemporaryClassLoader extends ClassLoader {
```

Comment:
I don't really like how this class is implemented, nor do I like the interface I created to avoid spying this class in the unit test.  I solicited feedback from a coworker who agrees and pointed me to a more straightforward example implementation.